modules/standard/BigInteger.chpl

```diff
@@ -2212,6 +2212,27 @@ module BigInteger {
 
 
   // divexact
+
+/*
+This function computes the quotient when a bigint variable *n* is divided by a
```

Comment:
While this is acceptable, I think it's a little verbose, given how many functions there will be to document. I would suggest spending less effort per each method, and keeping the documentation concise, e.g.

````rst
Computes ``n/d`` and stores result in the``bigint`` instance.

``divexact`` is optimized to handle cases where ``n/d`` results in an integer. 
When ``n/d`` does not produce an integer, this method may produce incorrect results.
````